packages/react-integration/cypress/integration/notificationdrawerbasic.spec.ts

```diff
@@ -5,6 +5,10 @@ describe('Notification Drawer Basic Demo Test', () => {
     cy.url().should('eq', 'http://localhost:3000/notification-drawer-basic-demo-nav-link');
   });
 
+  it('Verify x icon in header', () => {
```

Comment:
Can you also verify the interaction of closing the drawer?  I noticed the Example in the docs does not seem to be working.

Reply:
@tlabaj we don't open/close the drawer in the examples...only in the demo